src/player.h

```diff
@@ -180,6 +180,7 @@ class Player
 	HudElement* removeHud(u32 id);
 	void        clearHud();
 
+	u16 camera_mode_restrictions;
```

Comment:
initialize 0x01 | 0x02 | 0x04 here instead of the constructor function

Reply:
Done.